apps/token-manager/test/tokenmanager.js

```diff
@@ -384,25 +383,27 @@ contract('Token Manager', accounts => {
                 assert.equal(vRevokable, revokable)
             })
 
-            it('can start transfering on cliff', async () => {
-                await timetravel(cliff)
+            it('can start transferring on cliff', async () => {
```

Comment:
The `beforeEach()` could be modified as it is setting dates depending on the actual timestamp of the block, which is not used when mocking time.

I would also set an initial time right after initializing the contract so it is not 0.